omod/src/main/webapp/resources/htmlforms/019-HTSClientCard.xml

```diff
@@ -0,0 +1,680 @@
+                        <span style="display: none;">
+                                <encounterLocation default="629d78e9-93e5-43b0-ad8a-48313fd99117"
+                                                   order="629d78e9-93e5-43b0-ad8a-48313fd99117"/>
+                            </span>
+                    </td>
+                    <td>
+                        <label>Serial No:</label>
+                        <obs conceptId="1646" required="true"/>
+                    </td>
+                </tr>
+
+                <includeIf velocityTest="$patient.age &lt; 12">
+                    <tr>
+                        <td colspan="4">
+
+                            <label>Accompanied by:</label>
+                            <obs id="969" conceptId="969" answerConceptIds="970,971,165154,5620"
+                                 answerLabels="1 - Mother,2 - Father,3 - Both Parents, 4 - Other "/>
+                            <span >
+
+                                    <label>Specify</label>
+                                    <obs id="99369" conceptId="99369" class="horizontal-layout-options"/>
+
+                            </span>
+                        </td>
+                    </tr>
+                </includeIf>
+
+                </tbody>
+            </table>
+        </section>
+        <section>
+            <h3>Section B:</h3>
+            <table class="section-b">
+                <tbody>
+                <!--                    New row for HTS Delivery Model-->
+                <tr>
+                    <td style="width: 20%">
+                        <label>HTS Delivery Model:</label>
+                    </td>
+                    <td>
+                        <obs id="165171" conceptId="165171" answerConceptIds="99416,165160" answerLabels="1 - Health Facility(HF),2 - Community " style="radio" required="true"/>
+                    </td>
+                </tr>
+
+                <tr>
+                    <td>
+                        <label>HTS Approach:</label>
+                    </td>
+                    <td colspan="3">
+                        <obs id="br" conceptId="99462" answerConceptIds="162919,99459" answerLabels="1 - CICT/VCT,2 - PITC" style="radio" required="true"/>
+                    </td>
+                </tr>
+
+
+                <tr>
+                    <td>
+                        <label> Entry Point For Health Facility Testing:</label>
+                    </td>
+                    <td>
+                            <span >
+                                <obs conceptId="162925" id="162925"
+                                     answerConceptIds="160542,165179,165047,165048,165156,99593,164983,160456,165046,164984,165155,90015,90002"
+                                     answerLabels="1 - OPD,2 - Ward,3 - ART Clinic,4 - TB Clinic,5 - Nutrition unit,6 - Young Child Clinic (YCC),7 - ANC,8 - Maternity,9 - PNC,10 - Family Planning,11 - SMC,12 - STI Clinic,13 - Others(specify)"/>
```

Comment:
@dbaluku  Change "Others(specify)" to "Others (specify)" - notice a space before the parentheses